pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go

```diff
@@ -76,10 +76,6 @@ func NewRequestedToCapacityRatio(plArgs runtime.Object, handle framework.Framewo
 			resourceToWeightMap[v1.ResourceName(resource.Name)] = 1
 		}
 	}
```

Comment:
Could you add the same validations as the others?

Reply:
sure, will cc you then, this is in my TODO list.

Reply:
I think @pancernik can do it in his PRs for #90859